app/mobile/src/storybook/stories/Button/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { TouchableHighlight } from "react-native";
+
+export default function Button({ onPress, children }: {onPress: any, children: any}) {
```

Comment:
Is this an example component from the Storybook setup? Didn't we say we'll delete them to avoid confusion?

It's using things like `propTypes` here which is unnecessary since we are using TypeScript which will check the types for us at compile time rather than at run time.

Reply:
we will remove all this stuff.